rkt/image/dockerfetcher.go

```diff
@@ -50,7 +50,10 @@ type dockerFetcher struct {
 // ACI, then stores it in the store and returns the hash.
 func (f *dockerFetcher) GetHash(u *url.URL) (string, error) {
 	ensureLogger(f.Debug)
-	dockerURL := d2acommon.ParseDockerURL(path.Join(u.Host, u.Path))
+	dockerURL, err := d2acommon.ParseDockerURL(path.Join(u.Host, u.Path))
+	if err != nil {
+		return "", fmt.Errorf("invalid docker URL %q. Syntax docker://[REGISTRY_HOST[:REGISTRY_PORT]/]IMAGE_NAME[:TAG]", u)
```

Comment:
I thought about it, but I don't like the result: without `--debug`, we only print the first and last errors, that is: `run: empty Docker image reference` and I'd like to show the right syntax to the user. 

Maybe we can modify the error in docker2aci, although it doesn't make sense to include the `docker://` part because the function parses the part after the schema.